docs/explanation/planning.rst

```diff
@@ -2,36 +2,67 @@
 Planning
 ========
 
-The plan has a tree structure with three main sections: pre-upgrade steps,
-control-plane upgrade and data-plane upgrade.
+The plan has a tree structure with six main sections:
+
+* cloud pre-upgrade steps,
+* control-plane principal applications upgrade
+* control-plane subordinate applications upgrade
+* data-plane hypervisors upgrade
+* data-plane non-hypervisor principal applications upgrade
+* data-plane subordinate applications upgrade
+* cloud post-upgrade steps.
 
 .. code:: 
 
     Top-level plan
```

Comment:
Should this be changed to `Upgrade cloud from '{analysis_result.current_cloud_os_release}' to '{target}'`